openslides_backend/action/speaker/speak.py

```diff
@@ -0,0 +1,79 @@
+import time
+from typing import Iterable
+
+from ...models.models import Speaker
+from ...services.datastore.interface import GetManyRequest
+from ...shared.exceptions import ActionException
+from ...shared.interfaces import Event
+from ...shared.patterns import Collection, FullQualifiedId
+from ..base import Action, ActionPayload, DataSet, WriteRequestElement
+from ..default_schema import DefaultSchema
+from ..register import register_action
+
+
+@register_action("speaker.speak")
+class SpeakerSort(Action):
+    """
+    Action to let speakers speak.
+    """
+
+    model = Speaker()
+    schema = DefaultSchema(Speaker()).get_default_schema(
+        required_properties=["id"],
+        title="Speak schema",
+        description="Schema to let a speaker's speach begin.",
+    )
+
+    def prepare_dataset(self, payload: ActionPayload) -> DataSet:
```

Comment:
I suppose if a LOS is closed one cannot speak on it? If yes, this should be checked and tested here.

Reply:
Done.